vpdq/cpp/hashing/filehasher.h

```diff
@@ -18,27 +18,22 @@ namespace hashing {
  *
  * @param inputVideoFileName Input video's name
  * @param pdqHashes Vector which stores hashes
- * @param ffmpegPath Ffmpeg's path
  * @param verbose If produce detailed output for diagnostic purposes
  * @param secondsPerHash The time period of picking frames in vpdq
- * @param width Specified width of the input video
- * @param height Specified height of the input video
- * @param framesPerSec Frames per second of the input video
- * @param downsampleFrameDimension The down-scale dimensions for vpdq
- * @param programName The name of executable program which invokes the function
+ * @param downsampleWidth Width to downsample to before hashing. 0 means no
+ * downsample
+ * @param downsampleHeight Height to downsample to before hashing. 0 means no
+ * downsample
```

Comment:
Dumb question - is there ever a reason for these values to be different? PDQ wants a square aspect ratio at the end, of I fixed size IIRC

Reply:
Not sure. I'll read the PDQ doc and if not I'll change it.